packages/react-vapor/src/components/datePicker/DatePickerBox.tsx

```diff
@@ -124,7 +124,7 @@ export class DatePickerBox extends React.Component<IDatePickerBoxProps, any> {
 
     private getdateSelectionBoxes(): JSX.Element[] {
         return _.map(this.props.datesSelectionBoxes, (datesSelectionBox: IDatesSelectionBox) => {
-            const boxId: string = `${this.props.id}-${s.slugify(datesSelectionBox.title)}`;
+            const boxId: string = `${this.props.id}-${slugify(datesSelectionBox.title)}`;
```

Comment:
I think this lib is stronger for slugify https://github.com/simov/slugify

or create our own function to do it?
This one looks very good to me, but we'll have to write some tests :( https://gist.github.com/mathewbyrne/1280286#gistcomment-3753527